docs/en/query_language/functions/other_functions.md

```diff
@@ -4,6 +4,44 @@
 
 Returns a string with the name of the host that this function was performed on. For distributed processing, this is the name of the remote server host, if the function is performed on a remote server.
 
+## basename
+
+Extracts trailing part of a string after the last slash or backslash.
```

Comment:
Needs some comments about:
* whether it supports escaping and how it works.
* practical usecase of extracting filename from pathname.